### PR TITLE
MAUI-1725: Add an API reference link for Barcode and LinearGauge controls in UG

### PR DESCRIPTION
**Pull request by sivasuryanarayananv**
Add an API reference link for Barcode and LinearGauge controls in UG

### CODE REVIEW

File: MAUI/Barcode-Generator/getting-started.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ this.Content = barcode;
 
 ![.NET MAUI Barcode Generator Intialize Barcode](images/getting-started/maui-intialize-barcode.png)
 
-N> The default symbology of SfBarcodeGenerator is `Code128`.
+N> The default symbology of SfBarcodeGenerator is [`Code128`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.html).
```
Comment (vigneshrameshsync) [inline MAUI/Barcode-Generator/getting-started.md:97]: Add link for code 128

https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.Code128.html

File: MAUI/Barcode-Generator/one-dimensional-symbology.md [hunk 2]
```diff
@@ -379,9 +379,9 @@ As like `UPC-A`, the `UPC-E` symbology supports only numeric character. It is a
 
 ## EAN-13
 
-The `EAN-13`  standard is based on the `UPC-A` standard.. As like `UPC-A`, it supports only the numeric character. It encodes the 12 digits of input data with the check digit at its end.
+The [`EAN-13`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.EAN13.html) standard is based on the [`UPC-A`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.UPCA.html) standard.. As like [`UPC-A`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.UPCA.html), it supports only the numeric character. It encodes the 12 digits of input data with the check digit at its end.
```
Comment (vigneshrameshsync) [inline MAUI/Barcode-Generator/one-dimensional-symbology.md:382]: You have removed space

File: MAUI/Linear-Gauge/getting-started.md [hunk 3]
```diff
@@ -9,15 +9,15 @@ documentation: ug
 
 # Getting Started with .NET MAUI Linear Gauge
 
-This section explains the steps required to add the `.NET MAUI Linear Gauge` control and its elements such as scale, range, and pointer. This section covers only basic features needed to get started with Syncfusion linear gauge control.
+This section explains the steps required to add the [`.NET MAUI Linear Gauge`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html?tabs=tabid-1#Syncfusion_Maui_Gauges_SfLinearGauge_IsMirrored) control and its elements such as scale, range, and pointer. This section covers only basic features needed to get started with Syncfusion linear gauge control.
```
Comment (vigneshrameshsync) [inline MAUI/Linear-Gauge/getting-started.md:12]: Check the link

File: MAUI/Linear-Gauge/getting-started.md [hunk 4]
```diff
@@ -9,15 +9,15 @@ documentation: ug
...
 
 ## Creating an application using the .NET MAUI Linear Gauge
 
 * Create a new .NET MAUI application in the Visual Studio.
 
 * Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfLinearGauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges], and [Syncfusion.Maui.Core] then install that.
 
-* Import the control namespace `Syncfusion.Maui.Gauges` in XAML or C# code.
+* Import the control namespace [`Syncfusion.Maui.Gauges`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.html) in XAML or C# code.
```
Comment (vigneshrameshsync) [inline MAUI/Linear-Gauge/getting-started.md:20]: Here we are representing the namespace, not a control. So, please remove the link.

File: MAUI/Linear-Gauge/range.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ SfLinearGauge gauge = new SfLinearGauge();
...
...usion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_MidWidth) and [`EndWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndWidth) properties. To draw a line or rectangle, just [`StartValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartValue), [`StartWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_StartWidth), [`EndValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndValue) and [`EndWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_EndWidth) are enough - as like the above code snippet in [`Default Linear Gauge Range`]() topic. But to draw a concave and convex like shapes, [`MidWidth`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.LinearRange.html#Syncfusion_Maui_Gauges_LinearRange_MidWidth) property is needed. The below code snippet demonstrates how to bring a convex shape for a range.
```
Comment (vigneshrameshsync) [inline MAUI/Linear-Gauge/range.md:44]: Link is missing "[`Default Linear Gauge Range`]()"

File: MAUI/Linear-Gauge/scale.md [hunk 6]
```diff
@@ -302,7 +302,7 @@ SfLinearGauge gauge = new SfLinearGauge();
 
 ## Customize scale values
 
-Linear gauge allows you to display a set of values along with a custom scale based on your business logic by using the `GenerateVisibleLabels`,`ValueToFactor`, and `FactorToValue` callbacks.
+Linear gauge allows you to display a set of values along with a custom scale based on your business logic by using the [`GenerateVisibleLabels`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_GenerateVisibleLabels),[`ValueToFactor`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_ValueToFactor_System_Double_), and [`FactorToValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfLinearGauge.html#Syncfusion_Maui_Gauges_SfLinearGauge_FactorToValue_System_Double_) callbacks.
```
Comment (vigneshrameshsync) [inline MAUI/Linear-Gauge/scale.md:305]: provide space between "," and "ValueToFactor"

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: MAUI/Barcode-Generator/getting-started.md (hunk 1 above)
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/getting-started.md:97]: modified sir

File: MAUI/Barcode-Generator/one-dimensional-symbology.md (hunk 2 above)
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/one-dimensional-symbology.md:382]: The space is reflected in the correction part already. will correct this unwanted changes in future sir

File: MAUI/Barcode-Generator/two-dimensional-symbology.md [hunk 7]
```diff
@@ -55,22 +55,22 @@ A `QR Code` is a two-dimensional barcode that consists of a grid of dark and lig
 
 The data that can be stored in the QR code depends upon the following property:
 
-* `Error correction level`
-* `QR code version`
-* `Input mode`
+* [`Error correction level`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.ErrorCorrectionLevel.html)
+* [`QR code version`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.QRCodeVersion.html)
+* [`Input mode`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.QRCode.html#Syncfusion_Maui_Barcode_QRCode_InputMode)
```
[the remark this answers is not in the document]
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/two-dimensional-symbology.md:60]: Changed sir

File: MAUI/Barcode-Generator/two-dimensional-symbology.md [hunk 8]
```diff
@@ -55,22 +55,22 @@ A `QR Code` is a two-dimensional barcode that consists of a grid of dark and lig
...
 The property is used to define the size of the smallest line or dot of the barcode. If this property is not set, the size of the smallest dot of the barcode is calculated based on the available size.
 
 ### Error correction level
 
-The `ErrorCorrectionLevel` property employs error correction to generate a series of error correction code words that are added to the data code word sequence in order to enable the symbol to withstand damage without loss of data. By default, its value is set as `Auto`.
+The [`ErrorCorrectionLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.ErrorCorrectionLevel.html) property employs error correction to generate a series of error correction code words that are added to the data code word sequence in order to enable the symbol to withstand damage without loss of data. By default, its value is set as [`Auto`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.ErrorCorrectionLevel.html#Syncfusion_Maui_Barcode_ErrorCorrectionLevel_Auto).
```
[the remark this answers is not in the document]
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/two-dimensional-symbology.md:67]: Changed sir

File: MAUI/Barcode-Generator/two-dimensional-symbology.md [hunk 9]
```diff
@@ -113,12 +113,12 @@ The data can be read from the damaged image based on the error correction level.
...
 ### Input mode
 
-The `InputMode` property allows you to select a specific set of input characters. You can select the most suitable input mode. By default, its value is set as `Auto`.
+The [`InputMode`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.QRCode.html#Syncfusion_Maui_Barcode_QRCode_InputMode) property allows you to select a specific set of input characters. You can select the most suitable input mode. By default, its value is set as [`Auto`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.QRInputMode.html#Syncfusion_Maui_Barcode_QRInputMode_Auto).
 
 Numeric - supports from 0 to 9.
 AlphaNumeric - supports 0 to 9, A to Z, space, $, %, *, +, -,., /, :.
 Binary - supports Shift JIS characters.
-Auto - it calculates input mode automatically based on `CodeVersion` and `ErrorCorrectionLevel`.
+Auto - it calculates input mode automatically based on [`CodeVersion`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.QRCode.html#Syncfusion_Maui_Barcode_QRCode_CodeVersion) and [`ErrorCorrectionLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.ErrorCorrectionLevel.html).
```
[the remark this answers is not in the document]
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/two-dimensional-symbology.md:121]: Changed sir

File: MAUI/Linear-Gauge/getting-started.md (hunk 3 above)
Comment (sivasuryanarayananv) [reply MAUI/Linear-Gauge/getting-started.md:12]: Changed sir

File: MAUI/Linear-Gauge/getting-started.md (hunk 4 above)
Comment (sivasuryanarayananv) [reply MAUI/Linear-Gauge/getting-started.md:20]: Removed sir

File: MAUI/Linear-Gauge/scale.md (hunk 6 above)
Comment (sivasuryanarayananv) [reply MAUI/Linear-Gauge/scale.md:305]: provided sir

File: MAUI/Barcode-Generator/two-dimensional-symbology.md [hunk 10]
```diff
@@ -243,30 +243,30 @@ Type - Data Matrix supports the following data types:
...
 
-The `Size` depends upon the length and data type of provided input values.
+The [`Size`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.DataMatrix.html#Syncfusion_Maui_Barcode_DataMatrix_Size) depends upon the length and data type of provided input values.
 
 ### Encoding methods
 
-Data Matrix supports following `Encoding` types:
+Data Matrix supports following [`Encoding`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Barcode.DataMatrix.html#Syncfusion_Maui_Barcode_DataMatrix_Encoding) types:
 
 * Auto
```
[the remark this answers is not in the document]
Comment (sivasuryanarayananv) [reply MAUI/Barcode-Generator/two-dimensional-symbology.md:252]: Added the links for all the types sir

File: MAUI/Linear-Gauge/range.md (hunk 5 above)
Comment (sivasuryanarayananv) [reply MAUI/Linear-Gauge/range.md:44]: Since we can not find the exact topic, leave as it was before sir

**Comment by sivasuryanarayananv** [comment]
rebuild

**Comment by vigneshrameshsync** [comment]
rebuild

**Comment by sivasuryanarayananv** [comment]
rebuild

**Comment by sivasuryanarayananv** [comment]
rebuild

**Comment by sivasuryanarayananv** [comment]
rebuild

**Comment by sivasuryanarayananv** [comment]
rebuild

**Comment by sivasuryanarayananv** [comment]
rebuild

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Pull request merged**